style.css

```diff
@@ -88,6 +89,341 @@ a {
   padding: 5px;
 }
 
+.categories-list li {
+  font-weight: 400;
+  font-size: 1.2rem;
+  line-height: 3rem;
+  color: #fff;
+}
+
 .social-media ul li {
   padding: 5px;
 }
+
+#recentWorks {
+  display: grid;
+  height: auto;
+  grid-template-columns: 100%;
+}
+
+#recentWorks h2 {
+  color: #1c1a19;
+  font-weight: 400;
+  font-size: 36px;
+  border-bottom: 1px solid #212121;
+  align-items: center;
+  margin-top: 2em;
+  padding: 1em;
+}
+
+.imgHeader {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 2vh;
+}
+
+.imgHeader img {
+  width: 340px;
+}
+
+#seeProjects {
+  position: relative;
+  height: 100%;
+}
+
+#works h3 {
+  color: #091e42;
+  font-size: 29px;
+  font-weight: 400;
+  font-family: 'Crete Round', sans-serif;
+  margin: 0;
+}
+
+.introInfo {
+  color: #091e42;
+  font-size: 15px;
+}
+
+#works {
+  margin: 0 11vw;
+}
+
+#works ul {
+  display: flex;
+  padding-left: 0;
+}
+
+#works ul li {
+  border: 1px solid #8993a4;
+  background: #fff;
+  padding: 10px 12px;
+  color: #091e42;
+  margin: 3px;
+  list-style-type: none;
+}
+
+#works button {
+  display: flex;
+  background: #ff6b00;
+  color: #fff;
+  align-items: center;
+  padding: 12px;
+  justify-content: center;
+  width: 110px;
+  margin-top: 3px;
+  border: none;
+  cursor: pointer;
+}
+
+#cardWork button {
+  width: 100%;
+  position: absolute;
+  background-color: #ff6b00;
+  padding: 15px 0;
+  text-align: center;
+  margin-top: 2.5vh;
```

Comment:
Kindly avoid the use of vh value since it reacts unpredictably on different screen size.